dev-util/coverity/coverity-2017.07.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Static code analysis and dynamic code analysis tools"
+HOMEPAGE="https://www.coverity.com"
+SRC_URI="amd64? ( cov-analysis-linux64-${PV}.tar.gz )
+	x86? ( cov-analysis-linux-${PV}.tar.gz )"
+
+LICENSE="coverity-PLA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+L10N="ja ko zh-CN"
```

Comment:
Unused now? Also, for the future, please do not declare variables whose name collide with USE_EXPANDs. This may cause mayhem!

Reply:
This is right, fixed